script/hipify-inplace.sh

```diff
@@ -51,23 +61,37 @@ echo "Logs written to ${HIPIFY_LOG}"
 
 # Additional fixes for project-specific things and things hipify misses or gets
 # wrong.
-for src in ${srcs[@]}; do
-    sed -i 's@#include <hipblas.h>@#include <hipblas/hipblas.h>@' $src
-    sed -i 's@#include <hipsparse.h>@#include <hipsparse/hipsparse.h>@' $src
-    sed -i 's@#include <cuda_fp8.h>@#include <hip/hip_fp8.h>@' $src
-    sed -i 's@#include <cuda_bf16.h>@#include <hip/hip_bf16.h>@' $src
-    sed -i 's@\bDGL_USE_CUDA\b@DGL_USE_ROCM@g' $src
-    sed -i 's@\bCUB_VERSION\b@HIPCUB_VERSION@g' $src
-    sed -i 's@\bCUDART_ZERO_BF16\b@HIPRT_ZERO_BF16@g' $src
-    sed -i 's@\bCUDART_INF_BF16\b@HIPRT_INF_BF16@g' $src
-    sed -i 's@\bthrust::cuda::par@thrust::hip::par@g' $src
-    sed -i 's@\b__nv_fp8_e4m3\b@__hip_fp8_e4m3@g' $src
-    sed -i 's@\b__nv_fp8_e5m2\b@__hip_fp8_e5m2@g' $src
-    sed -i 's@\bCUBLAS_GEMM_DEFAULT_TENSOR_OP\b@HIPBLAS_GEMM_DEFAULT@g' $src
-    sed -i 's@\bcurand4\b@hiprand4@g' $src
-    # hipify uses the old one
-    sed -i 's@\bhip_bfloat16\b@__hip_bfloat16@g' $src
-    sed -i 's@\b__trap();@abort();@' $src
+for src in ${all_srcs[@]}; do
+    sed -i -f - $src <<-EOF
+        s@#include <hipblas.h>@#include <hipblas/hipblas.h>@
+        s@#include <hipsparse.h>@#include <hipsparse/hipsparse.h>@
+        s@#include <cuda_fp8.h>@#include <hip/hip_fp8.h>@
+        s@#include <cuda_bf16.h>@#include <hip/hip_bf16.h>@
+        s@#include <cub/cub.cuh>@#include <hipcub/hipcub.hpp>@
+        s@#include "hip/extension/@#include "./cuda/extension/@
+        s@#include "hip/cooperative_minibatching_utils.h"@#include "./cuda/cooperative_minibatching_utils.h"@
+        s@#include "hip/max_uva_threads.h"@#include "./cuda/max_uva_threads.h"@
+        s@\.\./hip/@@
+        s@\bDeviceFor::Bulk\b@Bulk@g
+        s@\bcub::@hipcub::@g
+        s@\bcuda::stream_ref@cuco::cuda_stream_ref@
+        s@\bcuda::::min@min@
+        s@\bcuda::proclaim_return_type@proclaim_return_type@
+        s@\bDGL_USE_CUDA\b@DGL_USE_ROCM@g
+        s@\bGRAPHBOLT_USE_CUDA\b@GRAPHBOLT_USE_ROCM@g
+        s@\bCUB_VERSION\b@HIPCUB_VERSION@g
+        s@\bCUDART_ZERO_BF16\b@HIPRT_ZERO_BF16@g
+        s@\bCUDART_INF_BF16\b@HIPRT_INF_BF16@g
+        s@\bthrust::cuda::par@thrust::hip::par@g
+        s@\b__nv_fp8_e4m3\b@__hip_fp8_e4m3@g
+        s@\b__nv_fp8_e5m2\b@__hip_fp8_e5m2@g
+        s@\bCUBLAS_GEMM_DEFAULT_TENSOR_OP\b@HIPBLAS_GEMM_DEFAULT@g
+        s@\bcurand4\b@hiprand4@g
+        s@\bhip_bfloat16\b@__hip_bfloat16@g `# hipify uses the old one`
+        s@\bnv_bfloat16\b@hip_bfloat16@g `# For some reason, some graphbolt dependencies need the old version. `
```

Comment:
You don't need to backtick-exec the comments now


```suggestion
        s@\bhip_bfloat16\b@__hip_bfloat16@g # hipify uses the old one
        s@\bnv_bfloat16\b@hip_bfloat16@g # For some reason, some graphbolt dependencies need the old version.
```